doc/classes/ConvexPolygonShape2D.xml

```diff
@@ -18,7 +18,7 @@
 			<argument index="0" name="point_cloud" type="PoolVector2Array">
 			</argument>
 			<description>
-				Currently, this method does nothing.
+				Based on the set of points provided, this creates and assigns the [member points] using the convex hull algorithm. Removing all unneeded points. (Convex hull is also available at [method Geometry.convex_hull_2d])
```

Comment:
Looks good, I would just slightly rephrase the last part (changes in bold):

> Based on the set of points provided, this creates and assigns the [member points] **property** using the convex hull algorithm**, r**emoving all unneeded points. See [method Geometry.convex_hull_2d] for details.

Reply:
Changes made.  :thumbsup: